tests/objects/test_frames.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+
+from encord.objects.frames import frames_class_to_frames_list, Range
+
+
+def test_frames_class_to_frames_list():
+    # Single frame
+    assert frames_class_to_frames_list(1) == [1]
```

Comment:
does it blow up with an empty list?

Reply:
It does not, it just returns an empty list. I added a test.